lib/main.dart

```diff
@@ -52,12 +62,13 @@ class Pangolin extends StatefulWidget {
   static OverlayState overlayState;
 
   static void setLocale(BuildContext context, Locale locale) {
+    EasyLocalization.of(context).locale = locale;
     _PangolinState state = context.findAncestorStateOfType<_PangolinState>();
```

Comment:
remove this whole part, as stated before easy_localization already stores the current locale.

Should remove line 55 and 56

Reply:
done